src/tool-colors.scss

```diff
@@ -1,3 +1,9 @@
 /* Tool specific colors */
 $assistants-blue: rgba(59, 139, 197, 1);
 $classmail-blue: rgba(59, 139, 197, 1);
+
+// From Nexus orange-dim used for accessibility in announcements
```

Comment:
Does this comment mean this is only for nexus? or will these colors be used across all tools?